docs/src/pages/components/badges/badges.md

```diff
@@ -66,3 +66,12 @@ import BadgeUnstyled from '@material-ui/unstyled/BadgeUnstyled';
 ```
 
 {{"demo": "pages/components/badges/UnstyledBadge.js"}}
+
+## Accessibility
+
+- `Badges` can be given accessible label using `aria-label` or `aria-labelledby`.
+- You can provide description using `aria-describedby`.
```

Comment:
I don't think this is relevant since props are spread anyway. We don't describe any attribute generally so this section is just bound to get outdated (e.g. when `aria-description` becomes stable).

Reply:
Should I remove line 73?

Reply:
Let's resolve https://github.com/mui-org/material-ui/pull/26009#discussion_r622068814 first.